lib/validataion/jsonSchemas.js

```diff
@@ -356,6 +356,7 @@ schemas[validationKeys.CONFIGURE] = {
 	'properties': {
 		'continueOnFatalError': {'type': 'boolean'},
 		'disallowCrashReports': {'type': 'boolean'},
+		'defaultTimeout': {}, // breaks when I put string/number, so it worked only when left empty
 		'logLevel': {'enum': ['silent', 'normal', 'verbose', 'debug']},
```

Comment:
Have you tried `'type': ['number', 'string']`?

Reply:
I suspect you may getting errors because same schema is used before and after this option is parsed, 'type': ['number', 'string'] might work too

Reply:
Thanks, I'll give it a try